cmd/updates.go

```
@@ -96,3 +82,22 @@ func checkUpdates(curVersion string, interval time.Duration, app *App) {
 		fnCheck()
 	}
 }
+
+func fetchAppUpdate(client *http.Client, url string) (*AppUpdate, error) {
+	resp, err := client.Get(url)
```

golangci-lint (2.12.2)

[error] (*net/http.Client).Get must not be called. use (*net/http.Client).Do(*http.Request)

(noctx)